airflow/requirements.txt

```diff
@@ -1,6 +1,6 @@
 calitp-data-infra==2024.1.3
 gusty==0.6.0
-pyairtable==1.0.0
+pyairtable==2.2.1
```

Comment:
The docs refer to a task that automatically deploys our additional Composer requirements, but I'm not sure if that's currently happening in practice upon merges to `main`. We should monitor [here](https://console.cloud.google.com/composer/environments/detail/us-west2/calitp-airflow2-prod/packages?authuser=1&project=cal-itp-data-infra) after merge to see if the referenced version number changes, and if it doesn't in about 15 minutes or so, modify the version number by hand and create a follow-up Issue to make sure package version changes get deployed successfully in the future.